plugin-opentelemetry/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "plugin-opentelemetry"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+alumet = { path = "../alumet" }
+anyhow = "1.0.88"
+log = "0.4.22"
+serde = { version = "1.0.210", features = ["derive"] }
+time = { version = "0.3.36", features = ["formatting"] }
+hyper = { version = "0.14", features = ["full"] }
+hyper-tls = "0.5"
+tokio = { version = "1.0", features = ["full"] }
+opentelemetry = { version = "*" }
+opentelemetry_sdk = { version = "*" }
+opentelemetry-otlp = { version = "*", features = ["grpc-tonic"]}
```

Comment:
The musl build is failing because (at least) one of the dependencies is looking for openssl. Probably hyper, so it will be resolved when you remove it from the dependency list. If opentelemetry does that too, you should tweak its features to use rustls instead openssl on musl target (see `plugin-influxdb` for example).